src/components/App.svelte

```diff
@@ -1,21 +1,35 @@
 <script>
+import { init as authenticate, me } from '../data/user'
+import { init as loadMessaging } from '../data/messaging'
+import { init as loadRequests } from '../data/requests'
 import Nav from './Nav.svelte'
 import Router from 'svelte-spa-router' // https://github.com/ItalyPaleAle/svelte-spa-router
 import { location } from 'svelte-spa-router'
 import Footer from './Footer.svelte'
 import routes from '../views/routes'
 import Bootstrap from './Bootstrap.svelte'
-import BrandOnly from './BrandOnly.svelte'
 import Error from './Error.svelte'
+
+const publicRoutes = ['/welcome/terms', '/welcome/privacy']
```

Comment:
Let's move the Terms to `/terms` and Privacy to `/privacy`. The nav for those pages should be a full nav if the user is logged in, but minimal otherwise.